provider/pkg/provider/remote/copyfile.go

```diff
@@ -29,7 +29,7 @@ func (c *CopyFile) Annotate(a infer.Annotator) {
 
 type CopyFileInputs struct {
 	Connection *Connection    `pulumi:"connection" provider:"secret"`
-	Triggers   *[]interface{} `pulumi:"triggers,optional" providers:"replaceOnDelete"`
+	Triggers   *[]interface{} `pulumi:"triggers,optional" provider:"replaceOnChanges"`
```

Comment:
Should we look for this and error during schema generation?

I don't know if it's idiomatic but typos are painful.

Reply:
Is there an existing way to warn, not error, in the code base already? I couldn't see anything.